### ISSUE TITLE
Update qtism package

### ISSUE THREAD

**Issue by xPaw** [Learnosity/learnosity-qti#3]
It's not really possible to install it at the moment:
```
  Problem 1
    - Installation request for learnosity/learnosity-qti ^0.5.0 -> satisfiable by learnosity/learnosity-qti[v0.5.0].
    - learnosity/learnosity-qti v0.5.0 requires qtism/qtism 0.9.11 -> no matching package found.
```

**Comment by stellalie** [comment]
Hi xPaw, I tried doing clean installation, ie. `git clone ..`, then `composer install` it should works out of the box. The package required is https://packagist.org/packages/qtism/qtism and I can see 0.9.11 is there. Not sure what's the issue is?

Btw, I just released `v0.6.0`.

**Comment by xPaw** [comment]
Not sure what happened last time, but it works now.
